jest.config.js

```diff
@@ -1,7 +1,11 @@
+/** @type {import('jest').Config} */
 module.exports = {
-  preset: "ts-jest",
+  bail: true,
   testEnvironment: "node",
-  testMatch: [
-    "**/src/zalopay/__tests__/**/*.test.ts"
-  ],
+  testTimeout: 60_000,
+  transform: {
+    "^.+\\.ts$": "@swc/jest"
+  },
+  moduleFileExtensions: ["ts", "js", "json"],
+  testMatch: ["**/*.test.ts"]
```

Comment:
@leonthaibao , @noradomi I see current config that jest executor only run the test files in the "**/src/zalopay/__tests__/" . This change impacts to us?

Reply:
@longxice I change the `testMatch` to match more test files, not only in `/src/zalopay/tests/`. So it's ok.